README.md

```diff
@@ -23,6 +24,21 @@ chardet.detectFile('/path/to/file', function(err, encoding) {});
 chardet.detectFileSync('/path/to/file');
 ```
 
+
+To return the full list of possible encodings:
+```javascript
+var chardet = require('chardet');
+chardet.detectAll(new Buffer('hello there!'));
```

Comment:
`Buffer.from`? https://nodejs.org/api/buffer.html#buffer_class_method_buffer_from_string_encoding

Using Buffer as constructor is deprecated AFAIK

Reply:
I copy/pasted from your example, but sure, I'll make this change. I'm not on the latest node, so I didn't know this has been deprecated. Thanks for the heads-up.